aio/content/examples/displaying-data/src/app/app-ctor.component.1.ts

```diff
@@ -8,11 +8,8 @@ import { Component } from '@angular/core';
     `
 })
 export class AppComponent {
-  title: string;
-  myHero: string;
 
-  constructor() {
-    this.title = 'Tour of Heroes';
-    this.myHero = 'Windstorm';
-  }
+  title: string = 'Tour of Heroes';
+  myHero: string = 'Windstorm';
```

Comment:
We can omit the type here, it will be inferred by TypeScript as `string`.
```suggestion
  title = 'Tour of Heroes';
  myHero = 'Windstorm';
```